scripts/camViewer

```diff
@@ -148,26 +148,27 @@ usage: $0 options
 start the viewer for controls cameras
 
 OPTIONS:
--c:            camera name as in camera list or gige #
--m:            bring up the edm screen
--r:            reboot the IOC
--l:            print list of cameras
--w #:         (wait for # hours to ask to renew, default 2 and 12 for GIGEs)
--u #:          update rate limit (default 5)
--H hutch:      use a specific hutches camviewer config file
--e|--enable:   enable camera ioc
--d|--disable:  disable camera ioc
+-c|--cam    :   camera name as in camera list or gige #
+-m|--main   :   bring up the edm screen
```

Comment:
```suggestion
-m|--main   :   bring up the edm screen
```
@silkenelson is that what the -m stands for?

Reply:
From what I've seen it is

Reply:
I guess it is... 
Kinda feel like most users would call the camviewer the "main" screen and that the option should be --edm or something though